spec/defines/files_spec.rb

```diff
@@ -65,4 +65,25 @@
     end
   end
 
+  context 'custom swapfilesize parameter with timeout' do
+    let(:params) do
+      {
+        #:ensure => "present",
+        :swapfile => "/mnt/swap.2",
+        :swapfilesize => '4.1 GB',
+        #:add_mount => true,
+        #:options => "defaults",
+        :timeout => 900,
+      }
+    end
+    it do
+      is_expected.to compile.with_all_deps
+    end
+    it do
+      is_expected.to contain_exec('Create swap file /mnt/swap.2').
+      with({"command"=>"/bin/dd if=/dev/zero of=/mnt/swap.2 bs=1M count=4402",
+       "creates"=>"/mnt/swap.2"})
```

Comment:
Can you add `"timeout" => 900` to this? Will actually test the exec has the right timeout.